03-Raster-Calculations-In-R.Rmd

```diff
@@ -143,97 +172,78 @@ CHM_HARV <- DSM_HARV - DTM_HARV
 
 #plot the output CHM
 plot(CHM_HARV,
-     main="NEON Canopy Height Model - Subtracted\n Harvard Forest") 
+     main="NEON Canopy Height Model - Manual Subtract\n Harvard Forest") 
 
 ```
 
-
-# Canopy Height Model Values
-
 Let's have a quick look at the range of values in our newly created Canopy
 Height Model (CHM).
 
 ```{r create-hist }
+#histogram of CHM_HARV
 hist(CHM_HARV,
      col="springgreen4",
      main="Histogram of NEON Canopy Height Model\nHarvard Forest",
```

Comment:
If we changed the previous title should this one be

Distribution of NEON Canopy Height Model Values ?